mithril-aggregator/src/artifact_builder/cardano_immutable_files_full.rs

```diff
@@ -24,7 +25,7 @@ use mithril_common::{
 pub enum CardanoImmutableFilesFullArtifactError {
     /// Protocol message part is missing
     #[error("Missing protocol message: '{0}'.")]
-    MissingProtocolMessage(String),
+    MissingProtocolMessage(StdError),
```

Comment:
This is maybe more straightforward:
```suggestion
    MissingProtocolMessage(Beacon),
```